pkg/morph/client/notary.go

```diff
@@ -151,6 +153,20 @@ func (c *Client) GetNotaryDeposit() (int64, error) {
 	return bigIntDeposit.Int64(), nil
 }
 
+// UpdateNotaryList updates list of notary nodes in designate contract. Requires
+// committee multi signature.
+func (c *Client) UpdateNotaryList(list keys.PublicKeys) error {
+	if c.notary == nil {
+		return errNotaryNotEnabled
```

Comment:
Is it expected to happen in app runtime? If not, I suggest to panic.

Reply:
It is not expected to happen in runtime.

This check applied in all notary related methods at `notary.go`. I don't mind to panic there but we should change it everywhere. Can you create an enhancement issue on this and then we'll fix it later?

Reply:
Good idea, check it out https://github.com/nspcc-dev/neofs-node/issues/445